ironfish-cli/src/commands/wallet/chainport/send.ts

```diff
@@ -84,7 +84,7 @@ export class BridgeCommand extends IronfishCommand {
 
     const networkId = (await client.chain.getNetworkInfo()).content.networkId
 
-    if (networkId !== TESTNET.id) {
+    if (networkId !== TESTNET.id && networkId !== MAINNET.id) {
       this.error(`Chainport transactions are only available on testnet.`)
```

Comment:
wouldn't the error message change here too?

Reply:
you're right, my bad